Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.cs

```diff
@@ -133,5 +132,34 @@ private static CompositeTransform GetAttachedCompositeTransform(UIElement elemen
 
             return result;
         }
+
+        private static EasingFunctionBase GetEasingFunction(InterpolationType interpolationType)
```

Comment:
Also, does it make sense to make this function public? This way animation extensions outside of the toolkit can take advantage of it.